sound/soc/sof/trace.c

```diff
@@ -13,6 +13,8 @@
 #include "sof-priv.h"
 #include "ops.h"
 
+#if !IS_ENABLED(CONFIG_SND_SOC_SOF_DISABLE_DMA_TRACE)
+
```

Comment:
Shouldn't we modify the Makefile: if DMA trace is not supported, we should not compile the module?

Reply:
++votes, full file is #ifdeffed out so better to disable in Makefile

Reply:
One more vote, should not compile the whole file once DMA trace is unselected.

Reply:
the trace is not a separate module, it's part of the core, I am not sure how to have a selective compilation.